packages/botonic-plugin-flow-builder/src/index.ts

```diff
@@ -180,10 +185,21 @@ export default class BotonicPluginFlowBuilder implements Plugin {
   }
 
   hasIntent(node: IntentNode, intent: string, locale: string): boolean {
-    const result = node.content.intents.find(
+    const intentFound = node.content.intents.find(
       i => i.locale === locale && i.values.includes(intent)
     )
-    return Boolean(result)
+    if (!intentFound) return false
+    return true
```

Comment:
Can be simplified and changed to (use of `some` instead of `find` to return a `boolean` directly):
```return node.content.intents.some(i => i.locale === locale && i.values.includes(intent))```

Reply:
👌 100% agree